ipympl/backend_nbagg.py

```diff
@@ -450,6 +455,11 @@ def handle_key(key):
 
         handle_key_press = handle_key_release = _handle_key
 
+    def _send_savefig_pdf(self):
+        buf = io.BytesIO()
+        self.figure.savefig(buf, format='pdf', dpi='figure')
+        self.send({'data': '{"type": "makepdf"}'}, buffers=[buf.getbuffer()])
```

Comment:
Changing this a little bit will allow generalizing to more than just PDFs.

1. Remove the arguments to savefig so that it follows the rcparams
2. check the rcparams format and use that to construct the mimetype
3. send that mimetype info along to the frontend

```suggestion
        self.figure.savefig(buf)
        format = matplotlib.rcParams['savefig.format']
        # some code intepreting the mimetype
        # e..g application/pdf, image/svg+xml, image/png etc
        self.send({'data': {"type": "download-fig", "format":format, "mimetype":mimetype}}, buffers=[buf.getbuffer()])
```